src/FSharp.Core/prim-types.fs

```diff
@@ -1754,12 +1754,12 @@ namespace Microsoft.FSharp.Core
                 checkType 0 [| rootType |]
 
             let arrayEqualityComparer<'T> er comparer =
-                let arrayEquals (er: bool) (iec: IEqualityComparer) (xobj: obj) (yobj: obj) : bool = 
+                let arrayEquals (er: bool) (iec: IEqualityComparer) (xobj: objnull) (yobj: objnull) : bool = 
                     match xobj, yobj with 
                     | null, null -> true
                     | null, _ -> false
                     | _, null -> false
-                    | (:? (obj array)      as arr1), (:? (obj array)      as arr2) -> GenericEqualityObjArray    er iec arr1 arr2
+                    | (:? (objnull array)      as arr1), (:? (objnull array)      as arr2) -> GenericEqualityObjArray    er iec arr1 arr2
```

Comment:
the alignment offends mine eyes :-)

Reply:
My lazy body relies on Fantomas too much, forgetting there are files which are ignored by it.
I will do something about it, I promise.